tensorboard/webapp/app_routing/store/app_routing_reducers.ts

```diff
@@ -23,6 +23,9 @@ const initialState: AppRoutingState = {
 
 const reducer = createReducer(
   initialState,
+  on(actions.discardDirtyUpdates, (state) => {
+    return {...state};
+  }),
```

Comment:
Please undo this change; this is basically noop and we do not need to make changes to the app_routing state based on this action at all.

Reply:
Done. Thanks!